### DISCUSSION TITLE
How to control header logo size

### DISCUSSION THREAD

**Discussion by cyberautomate** [chipzoller/hugo-clarity#236] [Q&A]
What is a or the preferred method of controlling the height and width of the site logo in the header? I have the logo in the header but after digging around in the layouts and .sass files I can't figure out how to make it larger. 

It is a large .png but hugo appears to be scaling it down.

**Comment by chipzoller** [comment]
I think that's correct based on the theme, right @onweru ?

**Reply by cyberautomate** [reply]
I did eventually figure out how to make this change for anyone else that wants to change this. The setting is in assets\sass\_components.sass. The default is 15rem

`&_brand
    img
      max-width: 20rem` 
      
How can I make this change without editing the _components.sass file? I don't want the change to be overwritten when I upgrade the theme with a new version.

**Reply by chipzoller** [reply]
We don't currently provide a configurable parameter to do this.

**Reply by OluchukwuON** [reply]
Was this eventually created?
